supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -602,7 +603,13 @@ class PipelinesApiAsyncBackendJobExecutionActor(override val standardParams: Sta
           && errorMsg.contains("Execution failed")
           && (errorMsg.contains("Localization") || errorMsg.contains("Delocalization"))) {
         s"Please check the log file for more details: $jesLogPath."
-      } else errorMsg
+      }
+      //If error code 10, add some extra messaging to the server logging
+      if (runStatus.errorCode.getCode.value == PapiErrorCode10) {
```

Comment:
I think you want an `else if` so as not to throw away the above message

Reply:
Yes!